src/components/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -16,11 +16,13 @@ import boxImage from "static/images/no-config-params.svg";
 import BooleanConfig from "./BooleanConfig";
 import TextAreaConfig from "./TextAreaConfig";
 
+import { generateIconImg } from "../../../pages/Models/Details/generators";
```

Comment:
This should be an absolute import.

```suggestion
import { generateIconImg } from "pages/Models/Details/generators";
```